src/libs/print_settings.c

```diff
@@ -1039,7 +1114,7 @@ gui_init (dt_lib_module_t *self)
   combo_idx = -1;
   n = 0;
 
-  dt_bauhaus_combobox_add(d->pprofile, _("none"));
+  dt_bauhaus_combobox_add(d->pprofile, _("server color managed"));
```

Comment:
What does that mean? Maybe reword that string?

Reply:
I want to say: "color management done on the server side", but with a smaller string.

Maybe "server side color management", it should fit into the dialog. Do you have a better proposal?

Reply:
Are users even aware that CUPS is a client/server system? Wouldn't they just assume it's a printer attached to their computer, and be confused by reading about a server being involved?

Maybe "color management in printer driver"?

Reply:
It fits, let's use this. thanks!